src/panels/config/zone/ha-config-zone.ts

```diff
@@ -219,8 +221,12 @@ export class HaConfigZone extends SubscribeMixin(LitElement) {
         .hass=${this.hass}
         .narrow=${this.narrow}
         .route=${this.route}
-        back-path="/config"
-        .tabs=${configSections.areas}
+        .backPath=${this._searchParms.has("historyBack")
+          ? undefined
+          : "/config"}
+        .tabs=${this._searchParms.has("noTabs")
+          ? configSections.areas.filter((page) => page.component === "zone")
+          : configSections.areas}
```

Comment:
I don't think we should do this... Is it that much of an distraction?

It is also weird that `noTabs` on the areas config page will give you the zone title 🙃

Reply:
No it's not that bad, could revert it if you prefer.

Just thought it made the interface a little cleaner when you're editing zones from the map, it just didn't seem like the tabs belonged there in that case. 

> It is also weird that noTabs on the areas config page will give you the zone title 

I don't understand how you got in this situation. If you're on the areas config page, isn't a different page being loaded that doesn't have this code, and this arg has no effect?

http://localhost:8123/config/areas/dashboard?noTabs=1

Reply:
In the interest of moving this forward, I dropped the changes to tabs, so now this just simply implements the back button logic.